notebooks/00_Setup.ipynb

```diff
@@ -47,8 +47,18 @@
    "id": "aa6bddcb",
```

Comment:
<p>Can you mention that this is an optional dependency?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/1029/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>